README.md

```diff
@@ -1 +1,39 @@
-# Cadastro-central
+# Central register
+
+## First steps
+### Database
+Before running our application, you have to create a postgres database locally with the following attributes:
+#### Server
+* Host: `localhost`
+* Port: `5432`
+* Database name: `central_register`
+#### Authentication
+* Authentication: `Database Native`
+* Username: `postgres`
+* Password: `postgres`
+
```

Comment:
The Lombok dependency used in the project requires a plugin in the IDE (at least for IntelliJ). Maybe it's a good ideia to say it here.

Reply:
wow I didn't know about this hahahah